azero/contracts/most-l2/lib.rs

```diff
@@ -339,14 +338,9 @@ pub mod most_l2 {
                 .transferred_value(amount_to_bridge)
                 .invoke()?;
 
-            let dest_token_address = self
-                .supported_pairs
```

Comment:
Do we want to remove this logic from contracts on AZERO side? https://github.com/Cardinal-Cryptography/most/pull/222/files#diff-ed93ab43297ff1acce2388424f1b571f305e5686aba3463a5d27d1b0393040eaR128

Reply:
This is removed here